generators/chipyard/src/main/scala/ChipTop.scala

```diff
@@ -45,13 +85,22 @@ abstract class BaseChipTop()(implicit val p: Parameters) extends RawModule with
 
 /**
  * A simple clock and reset implementation that punches out clock and reset ports with the same
- * names as the implicit clock and reset for standard Module classes. Reset is synchronous to
- * clock, which may not be a good idea to use for tapeouts.
+ * names as the implicit clock and reset for standard Module classes. Three basic reset schemes 
+ * a provided. See [[GlobalResetScheme]].
  */
 trait HasChipTopSimpleClockAndReset { this: BaseChipTop =>
 
   val (clock, systemClockIO) = IOCell.generateIOFromSignal(systemClock, Some("iocell_clock"))
-  val (reset, systemResetIO) = IOCell.generateIOFromSignal(systemReset, Some("iocell_reset"))
+  val (reset, systemResetIO) = p(GlobalResetSchemeKey) match {
+    case GlobalResetSynchronous  =>
+      IOCell.generateIOFromSignal(systemReset, Some("iocell_reset"))
+    case GlobalResetAsynchronousFull =>
+      IOCell.generateIOFromSignal(systemReset, Some("iocell_areset"), abstractResetAsAsync = true)
```

Comment:
```suggestion
      IOCell.generateIOFromSignal(systemReset, Some("iocell_reset"), abstractResetAsAsync = true)
```

Unless we're going to suggestName `reset` to `areset` we probably want to keep the IO cell naming convention of `iocell_ + <signal name>`

Reply:
sounds good.